CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ set(PROJECT_DESCRIPTION "")
 set(CMAKE_CXX_STANDARD 17)
 set(PROJECT_USE_CMAKE_EXPORT TRUE)
 set(CXX_DISABLE_WERROR ON)
+set(USE_ROS2 OFF)
```

Comment:
Wouldn't `option` be appropriate instead of `set` for variables that are supposed to be set by the user?